plonky2/src/fri/oracle.rs

```diff
@@ -55,7 +55,7 @@ impl<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const D: usize>
 {
     /// Creates a list polynomial commitment for the polynomials interpolating the values in `values`.
     pub fn from_values(
-        values: Vec<PolynomialValues<F>>,
+        values: &[PolynomialValues<F>],
```

Comment:
I think it's better to keep a `Vec<_>` here so that it's explicit that this function requires cloning the data.

Reply:
You know what, I think you're right. I made an attempt to take this further but it's really built to take ownership of the data. If we want less cloning we should look elsewhere. Sigh.

Reply:
@Nashtare Unless you think differently I'll revert this change and just remove the TODO.